src/components/hooks/use-cookie-banner.ts

```diff
@@ -38,10 +38,8 @@ export const useCookieBanner = () => {
             if (is_eu && !tracking_status) setShouldShow(true)
 
             if (allow_tracking) {
```

Comment:
```suggestion
            if (allow_tracking) {
```
if (allowTracking && document.readyState === 'complete') {
    setGTMData({ event: 'allow_tracking' });
  }

Reply:
Ohhh yes I will update that @habib-deriv thanks for highlighting 🙇